### PR TITLE
docs: update changelog for 1426

### PR DESCRIPTION
**Pull request by paul-schaaf**
#1426 may break some programs

### CODE REVIEW

**Comment by paul-schaaf** [comment]
asking for review because Im not 100% sure it will break some programs

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ incremented for features.
 ### Breaking
 
 * lang: Enforce that the payer for an init-ed account be marked `mut` ([#1271](https://github.com/project-serum/anchor/pull/1271)).
+* lang: all error code is now in the error module ([#1426](https://github.com/project-serum/anchor/pull/1426)).
```
Comment (armaniferrante) [inline CHANGELOG.md:21]:
```suggestion
* lang: All error codes are now in the error module ([#1426](https://github.com/project-serum/anchor/pull/1426)).
```

**Comment by armaniferrante** [comment]
> asking for review because Im not 100% sure it will break some programs

It is technically a breaking change for those bold enough to depend on a module named __private.

**Comment by paul-schaaf** [comment]
I meant `code` (as in lines of code) not `codes` btw :P

**Comment by paul-schaaf** [comment]
Id say technically breaking is still breaking so we should add it

**Comment by armaniferrante** [comment]
> Id say technically breaking is breaking so we should add it

sgtm!

**Pull request merged**